### PR TITLE
Add Mesh into builder tools and showcases

### PR DESCRIPTION
**Pull request by jinglescode**
## Showcase addition

* Name: Mesh
* Description: An Open-Source library by Martify Labs with the purpose of making building dApps on Cardano easier than ever. The Mesh library is engineered to simplify the development and implementation of blockchain technology, so you can focus on your project.
* Link: [https://mesh.martify.io/](https://mesh.martify.io/)
* Source: [https://github.com/MartifyLabs/mesh](https://github.com/MartifyLabs/mesh)
* Tags:
  * `educational`
  * `opensource`
  * `minting`

## Builder tools addition

* Name: Mesh
* Description: An Open-Source library by Martify Labs with the purpose of making building dApps on Cardano easier than ever. The Mesh library is engineered to simplify the development and implementation of blockchain technology, so you can focus on your project.
* Link: [https://mesh.martify.io/](https://mesh.martify.io/)
* Get started: [https://mesh.martify.io/guides](https://mesh.martify.io/guides)
* Tags:
  * `javascript`
  * `typescript`
  * `serialization`
  * `sdk`
  * `plutus`

### CODE REVIEW

File: src/data/builder-tools.js
Comment (fill-the-fill) [inline src/data/builder-tools.js:770]:
```suggestion
    getstarted: null,
```

There is a specific section in get started on developer portal dedicated to builder tools get started tutorials. The format is in markdown. Feel free to add your guide there as well, otherwise please write null in that field 😊

**Review by fill-the-fill — CHANGES_REQUESTED** [review]

File: src/data/builder-tools.js
Comment (jinglescode) [reply src/data/builder-tools.js:770]: A get started page on the developer portal is a work in progress. Will update this field when it is done. For now, setting `null` for `getstarted`.

**Comment by fill-the-fill** [comment]
Few more things I've noticed: let's keep it consistent and submit image in `.png` format and also I believe submitting your project as a builder tool is enough, no need to use it as a showcase in my opinion.

**Comment by jinglescode** [comment]
> Few more things I've noticed: let's keep it consistent and submit image in `.png` format and also I believe submitting your project as a builder tool is enough, no need to use it as a showcase in my opinion.

Have made the changes as requested on the PNG image. As for showcase, I would like to propose keeping it in showcase too as the [Playground](https://mesh.martify.io/apis/transaction) is highly interactive to showcase what Cardano can do. Both the package and the web demo is a valuable educational tool as currently, we have 70 student developers learning to build applications on Cardano with it. Also, we open source the best engineering practices and the code is well engineered. 

Let me know what you think, if you wish for it to be removed from the showcase, I will do that.

**Review by rphair — COMMENTED** [review]
FYI @jinglescode my mention of this PR in the issue directly above (https://github.com/cardano-foundation/developer-portal/issues/742#issuecomment-1252328743) is to get some clarity on when `getstarted:` is allowed to link to external web pages:as it does in some cases.  These may be artefacts that we have no choice but to keep.  In the meantime your setting `getstarted:` to `null` is probably the best thing to do.

> As for showcase, I would like to propose keeping it in showcase too as the [Playground](https://mesh.martify.io/apis/transaction) is highly interactive to showcase what Cardano can do.

I would generally be happy to do this, if linking distinctly to two different but affiliated web pages, but in this case:

 - The precedent for a code proving / demonstration / learning environment is to put it in Builder Tools alone (example, the Marlowe Playground: https://developers.cardano.org/tools?tags=marlowe).
 - The use of the term "Showcase" is as a showcase of projects & implementations, not to "showcase" developer features & means of programming... all of which are learning environments of use to experienced Builders in creating systems & using Tools (hence the latter are consistently categorised as "Builder Tools").

So I would say this is ready to go as a Builder Tool but I agree with @fill-the-fill about the Showcase entry being inappropriate.

**Comment by jinglescode** [comment]
Thanks @fill-the-fill and @rphair. I have removed Mesh from showcases. Noted on that "to showcase projects & implementations", will include it when we are ready to feature them.

**Review by rphair — CHANGES_REQUESTED** [review]
thanks @jinglescode ... all these changes are ready to go, but please first remove `yarn.lock` which somehow made it into your PR manifest 😁

**Comment by jinglescode** [comment]
> thanks @jinglescode ... all these changes are ready to go, but please first remove `yarn.lock` which somehow made it into the "Files changed" for this PR 😁

Done! Deleted `yarn.lock`

**Review by rphair — APPROVED** [review]
Builds & runs fine with current `staging` ✔️

**Review by fill-the-fill — APPROVED** [review]

**Comment by jinglescode** [comment]
Hi @fill-the-fill, are we just waiting for 2 more reviewers to approve this PR? Or is there an action needed on my part to proceed?

**Review by rdlrt — APPROVED** [review]

**Comment by rphair** [comment]
builds OK with `staging` after resolving merge conflict ✔️

**Pull request merged**
